tests/guarded_transmute_vec_pedantic/mod.rs

```diff
@@ -1,20 +1,21 @@
 #![cfg(feature = "std")]
 
 
-use safe_transmute::{ErrorReason, GuardError, Error, guarded_transmute_vec_pedantic};
+use safe_transmute::{ErrorReason, GuardError, Error, PedanticGuard};
+use safe_transmute::base::guarded_transmute_vec;
 use self::super::LeToNative;
 
 
 #[test]
 fn too_short() {
     unsafe {
-        assert_eq!(guarded_transmute_vec_pedantic::<u16>(vec![]),
+        assert_eq!(guarded_transmute_vec::<u16, PedanticGuard>(vec![]),
```

Comment:
Module name doesn't match tested funxion – are we letting this slide 'till the test reorganisation?